generators/typescript/sdk/client-class-generator/src/endpoints/default/endpoint-response/GeneratedThrowingEndpointResponse.ts

```diff
@@ -370,6 +374,16 @@ export class GeneratedThrowingEndpointResponse implements GeneratedEndpointRespo
             : name.name.camelCase.safeName;
     }
 
+    public getDefaultValue(page: RequestProperty): number {
+        if (page.property.valueType.type === "primitive") {
+            const v2PrimitiveType = page.property.valueType.primitive.v2;
+            if (v2PrimitiveType?.type === "integer") {
+                return v2PrimitiveType.default ?? 0;
+            }
+        }
+        return 0;
```

Comment:
Can you set the default to 1, otherwise we won't be able to merge since this would be a breaking change for users?